src/server/controllers/progress-bar.js

```diff
@@ -0,0 +1,10 @@
+import dbquery from './dbaccess.js';
```

Comment:
I should have caught this before I merged... I'm trying to run the server and it fails on this file. 

* there is no `dbaccess.js`
* the function below is malformed
* i can't find where this file is being imported and used